### PR TITLE
7 post article comments

### PR DESCRIPTION
**Pull request by IsmailMohamed125**
I've left a comment on my second bad post-request test, I would appreciate feedback on this!

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -205,7 +204,75 @@ describe("Articles Endpoint", () => {
...
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send({})
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Bad request");
+        });
+    });
+    test("POST:400 - Responds with an error when attempting to make a POST request with valid fields but the value of a field is invalid", () => {
+      // Comment for NC staff members: Since PSQL is doing type coercion even if i send a non-string this test will fail without the use of an external library. Also, the other way i could check is converting the string values i get from req.body to another type in an if statement like if(Number(usermame)) then reject there but I would have to check mutiple data types. Please suggest another way if you can but for now i will use a library called express-validator I will only use it for this case.
```
Comment (nc-mark-t) [inline __tests__/app.test.js:242]: This is probably not worth testing for. Could a comment just be numbers? Probably... and you can be pretty confident it's not going to break anything too. Unlike the other way around.

File: app.js [hunk 2]
```diff
@@ -8,6 +8,8 @@ const {
   serverErrorHandler,
 } = require("./controllers/error.controller");
 
+app.use(express.json());
```
Comment (nc-mark-t) [inline app.js:11]: 👍  - Good time to introduce this

File: models/articles.js [hunk 3]
```diff
@@ -46,4 +48,21 @@ function selectComments(article_id) {
     });
 }
 
-module.exports = { selectArticles, selectArticle, selectComments };
+function insertComment(newComment, article_id) {
+  const formattedComment = prepareNewComment(newComment, article_id);
+  const queryString = format(
```
Comment (nc-mark-t) [inline models/articles.js:53]: Any reason you're using `pg-format` here and not just `$1` from `pg`?

**Review by nc-mark-t — COMMENTED** [review]
Good work 👍 Using the express-validator might be overkill but I think as long as you know how to do it manually it's fine.

File: models/articles.js (hunk 3 above)
Comment (IsmailMohamed125) [reply models/articles.js:53]: No reason, I forgot that was just one comment being added so no need to use pg-format will update

**Pull request merged**